docs/spec/dataclasses.rst

```diff
@@ -526,6 +529,52 @@ This includes, but is not limited to, the following semantics:
   dataclass must be explicitly annotated as e.g. ``x: ClassVar[Final[int]] =
   3``.
 
+Converters
+^^^^^^^^^^
+
+The ``converter`` parameter can be specified in a field definition to provide a callable
+used to convert values when assigning to the associated attribute. This feature allows for automatic type
```

Comment:
Here again, is "associated attribute" meaningfully different from just saying "field"?